core/templates/pages/exploration-editor-page/modal-templates/lost-changes-modal.template.html

```diff
@@ -13,8 +13,7 @@ <h3>Error Loading Exploration</h3>
     paste these changes before discarding them.
   </p>
 
-  <div class="oppia-lost-changes" ng-bind-html="lostChangesHtml">
-  </div>
+  <changes-in-human-readable-form lost-changes="lostChanges" />
```

Comment:
Please use explicit tags to close: `<...></...>`

Reply:
Done.